internal/command/command.go

```diff
@@ -121,7 +121,12 @@ func cleanTargetDirs(ctx Context, image containerImage) error {
 // prepareIcon prepares the icon for packaging
 func prepareIcon(ctx Context, image containerImage) error {
 	if !ctx.NoProjectUpload {
-		if _, err := os.Stat(ctx.Icon); os.IsNotExist(err) {
+		iconPath := ctx.Icon
+		if ctx.Icon[0] != os.PathSeparator {
```

Comment:
This seems too simple - what about "./" or "../" - maybe it's need for another absolute check?

Reply:
In this case, we need to build an absolute path. I guess I could use filepath.IsAbs here.

Reply:
That is what I meant, sorry - my examples are bad. An absolute path is not defined as starting with the path separator on all OS.